tables/uz-g1.utb

```diff
@@ -50,15 +52,39 @@ uplow \x049A\x049B 13456      #қ
 uplow \x040E\x045E 1236       #ў
 
 # Equivalent to the additional braille adaptations.
-word sh 156
-word ch 12345
-word o’ 1236
-word g‘ 12456
+always sh 156
+always ch 12345
+always o’ 1236
+always o‘ 1236
+always o` 1236
+always g‘ 12456
+always g’ 12456
+always g` 12456
+
+# Symbol fixes
+math % 3456-245-356
+math ° 46-356
+punctuation ( 2356
+punctuation ) 2356
+punctuation { 126
+punctuation } 345
+noback punctuation [ 126
+noback punctuation ] 345
+sign @ 4-1
+sign \x2018 3
+sign \x0301 4 # acute accent
+
+noback pass2 @46-46-46 @46
+attribute roman IVXLCDM
 
 include ru-litbrl.ctb
 
 # Remove indicators before capital letters and between cyrillic and latin letters.
+attribute roman IVXLCDM
```

Comment:
Seems this line is duplicated?

Reply:
That attribute was kept twice deliberately, one before and one after including ru-litbrl.ctb.
I observed a malfunction regarding this, as you saw I have a rule for removing some capital letter signs:
`noback pass3 [@46]!%roman ?` -> If not a roman capital letter is encountered after capital sign, remove the pattern 46 (because they don't use capital letters, they are using it only for roman numbers).
Now the problem is:
- If I keep only the attribute definition above the inclusion, "VI" is translated correctly with capital sign in front of it, but "IV" has no begcapsletter.
- In the other hand, if I keep only the one beneath the inclusion, "VI" has no capital sign, but "IV" has.

The odd thing is that attribute "roman" remains the same, no clue is showing that ru-litbrl.ctb changes something (checked with lou_trace).
If I keep both definitions, everything works fine...

Reply:
I'll check what's going on.